cnd/cnd.toolchain.ui/src/org/netbeans/modules/cnd/toolchain/ui/compiler/CodeAssistancePanelController.java

```diff
@@ -26,12 +26,14 @@
 import org.openide.util.HelpCtx;
 import org.openide.util.Lookup;
 
-@OptionsPanelController.SubRegistration(
-    id=CndUIConstants.TOOLS_OPTIONS_CND_CODE_ASSISTANCE_ID,
-    location=CndUIConstants.TOOLS_OPTIONS_CND_CATEGORY_ID,
-    displayName="#TAB_CodeAssistanceTab", // NOI18N
-    position=300
-)
+// Overriden with a manually generated layer.xml file.
+// See https://issues.apache.org/jira/browse/NETBEANS-6372
+//@OptionsPanelController.SubRegistration(
+//    id=CndUIConstants.TOOLS_OPTIONS_CND_CODE_ASSISTANCE_ID,
+//    location=CndUIConstants.TOOLS_OPTIONS_CND_CATEGORY_ID,
+//    displayName="#TAB_CodeAssistanceTab", // NOI18N
+//    position=300
+//)
```

Comment:
I suggest to remove the commented out sections and add a comment in the `layer.xml` that the definition was moved to the layer file because the `weight` attribute needs to be defined.

Reply:
Agreed. Thanks!